tests/publiccloud/migration.pm

```diff
@@ -118,6 +119,17 @@ sub run {
     record_info('Migrated Version', $migrated_version);
 
     die("Wrong version: expected: " . $target_version . ", got " . $migrated_version) if ($migrated_version ne $target_version);
+
+    if ($add_boot) {
+        record_info('system reboots after succesfull migration');
+        my ($shutdown_time, $startup_time) = $instance->softreboot(
+            timeout => get_var('PUBLIC_CLOUD_REBOOT_TIMEOUT', 400),
+            ignore_wrong_pubkey => 1
```

Comment:
you should not use it in this case , we not expecting "wrong pubkey" error here , so if it will happen we should fail

Reply:
done